docs/recipes/custom-transformer.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Custom transformer
+---
+
+## Use case
+
+Sometimes the default response serializer isn't what you need and you're looking for a way to overwrite the default serialization for a response.
+
+For example, the default `JSON.stringify` implementation doesn't serialize `BigInt` values correctly, so you need to overwrite it with another serializer.
```

Comment:
Let's paraphrase it as a tutorial setting:

```
In this recipe we are going to write a custom response transformer that would serialize a `BigInt` values that cannot be otherwise transformed using `JSON.stringify`.
```